patches/ethers+5.0.0-beta.190.patch

```diff
@@ -0,0 +1,50 @@
+diff --git a/node_modules/ethers/dist/ethers.umd.js b/node_modules/ethers/dist/ethers.umd.js
+index fa28443..8b638a7 100644
+--- a/node_modules/ethers/dist/ethers.umd.js
++++ b/node_modules/ethers/dist/ethers.umd.js
+@@ -1,3 +1,4 @@
++var RNScrypt = require("react-native-scrypt").default;
+ (function (global, factory) {
+ 	typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports) :
+ 	typeof define === 'function' && define.amd ? define(['exports'], factory) :
+@@ -16444,23 +16445,23 @@
+
+ 	    const lib = {
+ 	        scrypt: function(password, salt, N, r, p, dkLen, progressCallback) {
+-	            return new Promise(function(resolve, reject) {
+-	                let lastProgress = 0;
+-	                if (progressCallback) { progressCallback(0); }
+-	                _scrypt(password, salt, N, r, p, dkLen, function(error, progress, key) {
+-	                    if (error) {
+-	                        reject(error);
+-	                    } else if (key) {
+-	                        if (progressCallback && lastProgress !== 1) {
+-	                            progressCallback(1);
+-	                        }
+-	                        resolve(new Uint8Array(key));
+-	                    } else if (progressCallback && progress !== lastProgress) {
+-	                        lastProgress = progress;
+-	                        return progressCallback(progress);
+-	                    }
+-	                });
+-	            });
++	        	return new Promise(function(resolve, reject) {
++								let lastProgress = 0;
```

Comment:
the indentation looks wrong

Reply:
agree, but I think patcher recognizes it wrong, didn't put too much effort in trying to fix this, but will check once again

Reply:
looks like it was replaced with tabs instead of spaces

Reply:
was a tricky one, starts with tab, ends with spacing :) copying over exact indentation worked, thanks 👍